blender_3dmigoto.py

```diff
@@ -2567,13 +2571,8 @@ class Export3DMigoto(bpy.types.Operator, ExportHelper):
     """Export a mesh for re-injection into a game with 3DMigoto"""
     bl_idname = "export_mesh.migoto"
     bl_label = "Export 3DMigoto Vertex & Index Buffers"
-
-    filename_ext = '.vb0'
-    filter_glob: StringProperty(
-            default='*.vb*',
-            options={'HIDDEN'},
-            )
-
+    filename_ext = '.ini'
```

Comment:
Hmm... not sure about this. I guess it does get you protection against overwriting the ini file, which is probably more important than protection against overwriting the .vb files... Was that your reasoning for this change, or something else?